### ISSUE TITLE
React 19 Support

### ISSUE THREAD

**Issue by steve-marmalade** [quillforms/quillforms#195]
**Describe the bug**
React 19 has [removed `findDOMNode`](https://github.com/facebook/react/issues/28926), which this library appears to depend on.

**To Reproduce**
1. Create a new NextJS project using the latest canary: `npx create-next-app -e reproduction-template`
2. Install `quillforms`
3. Try to instantiate a `Form`

**Expected behavior**
Expected this library to work with React 19

**Screenshots**
![image](https://github.com/user-attachments/assets/41fa5e93-55c1-4184-86ff-b6fea9b2a6bc)

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser Chrome

**Comment by Orescout** [comment]
any updates or workarounds or fixes on this @steve-marmalade @mdmag @ah2048 ? Hitting similar error because of this and can't get through.

**Comment by ecaglar** [comment]
@boyewale I checked source code and `findDOMNode` is not in `use-handle-focus` anymore. any idea?

**Comment by ecaglar** [comment]
@boyewale I couldnt find it unfortunately. It shows different code. Is there any way that you can share it so that i can just use?

**Comment by mdmag** [comment]
I will try to have a new release tomorrow to fix this issue.

**Comment by gracecodes144** [comment]
Blocked by this it would be awesome to get the new release! Thanks so much for your work!

**Comment by nicolo-paganin** [comment]
any news on react 19 support?

**Comment by beastafk** [comment]
Also it seems like in a lot of places the peerDeps are pinned to React(Dom) 18:

```
"peerDependencies": {
	"react": "^18.0.0",
	"react-dom": "^18.0.0"
},
```

Are there any plans to upgrade everything to React 19? If it's too much effort, maybe we can have some instructions on how to build and run the project as a standalone app so the community can (try to) help.

**Comment by mdmag** [comment]
I will try my best today.

**Comment by nicolo-paganin** [comment]
any news on this?

**Comment by natcohen** [comment]
When trying the example in React 19, I get the following error: `Uncaught TypeError: Cannot read properties of undefined (reading 'getBlockTypes')` is it related to React or is it another issue?
